tools/cli/test/data/changelog/same-api-version/output/2023-01-01_2023-02-01.json

```diff
@@ -0,0 +1,364 @@
+[
```

Comment:
is this the test output or the expected output?

Reply:
if test output, can we add to git ignore

Reply:
no, there are the files that I downloaded (manually) from evergreen and are compared to the ones generated by the e2e test